python_modules/libraries/dagster-dg/dagster_dg/context.py

```diff
@@ -334,6 +336,20 @@ def project_python_executable(self) -> Path:
             )
         return self.root_path / get_venv_executable(Path(".venv"))
 
+    @cached_property
+    def build_config_path(self) -> Path:
+        return self.root_path / "build.yaml"
```

Comment:
`registry.yaml` is now this (more expansive) config, I imagine?

Reply:
we should discuss that... i was imagining augmenting build.yaml rather than a separate registry.yaml?